src/app/components/core/plugin-modal/plugin-modal.component.ts

```diff
@@ -101,7 +102,12 @@ export class PluginModalComponent implements OnInit, OnChanges {
           } else if (error.status === 404) {
             this.alertService.error('Make sure package repository is configured / added in FogLAMP');
           } else {
-            this.alertService.error(error.statusText);
+            let errorText = error.statusText;
+            if (typeof error.error.link === 'string') {
+              errorText += ' <a href=' + this.PACKAGE_LOG_URL + error.error.link + '>'
```

Comment:
you can use string interpolation here. 
e.g. errorText += ` <a href= ${this.PACKAGE_LOG_URL}${error.error.link}>${error.error.link}</a>`